instrumentation/p6spy/src/main/java/brave/p6spy/TracingJdbcEventListener.java

```diff
@@ -35,12 +38,14 @@
 
   @Nullable final String remoteServiceName;
   final boolean includeParameterValues;
+  final boolean includeAffectedRowsCount;
   final P6LogLoadableOptions logOptions;
 
   TracingJdbcEventListener(@Nullable String remoteServiceName, boolean includeParameterValues,
-    P6LogLoadableOptions logOptions) {
+    boolean includeAffectedRowsCount, P6LogLoadableOptions logOptions) {
```

Comment:
Is this a breaking change?

Reply:
Thanks for pointing that out. I´ve added a version of the constructor that has the old signature, which delegates to the new one.